{{cookiecutter.project_slug}}/tests/test_{{cookiecutter.project_slug}}.py

```diff
@@ -15,62 +15,55 @@
 
 import {{ cookiecutter.project_slug }}
 {%- if cookiecutter.use_pytest == 'y' %}
-import os
+
 from fastapi.testclient import TestClient
 
 from {{ cookiecutter.project_slug }}.create_app import app
-import {{ cookiecutter.project_slug }}.core.env as environment
-
-SERVICE_NAMESPACE = environment.SERVICE_NAMESPACE
 
 
 class Test{{ cookiecutter.project_slug|title }}():
 
     @classmethod
-    def setup_class(self):
+    def setup_class(cls):
         """Set up test fixtures."""
-        self.app = TestClient(app)
-        self.endpoint_prefix = ''
-
-        if 'service_namespace' in os.environ and \
-                os.environ['service_namespace'] == SERVICE_NAMESPACE:
-            self.endpoint_prefix = f'{SERVICE_NAMESPACE}'
+        cls.app = TestClient(app)
+        cls.endpoint = 'test-service'
 
-    def teardown_class(self):
+    def teardown_class(cls):
```

Comment:
It is not a class function, why should it be cls? Maybe you need toa dd @classmethod